packages/cli/src/utils.ts

```diff
@@ -146,6 +146,15 @@ export const  getConfigWithChecks = async (
 
   validateLocalizableResources(config);
 
+  const googlePkg = config.solana_mobile_dapp_publisher_portal.google_store_package;
+  if (googlePkg != "" && googlePkg.length > 0) {
+    const pkgCompare = new RegExp("[a-zA-Z0-9_]+(\\.[a-zA-Z0-9_]+)+").exec(googlePkg);
+
+    if (pkgCompare == null || pkgCompare?.length == 0) {
```

Comment:
is the `?` neccessary? The first part of the test excluded the possibility that `pkgCompare` is `null`.

Alternatively, this test could probably be simplified to `if (!pkgCompare?.length) {` - both null and 0-length would be false-y.

Reply:
One of these days I'm gonna get TS/JS syntax